openmdao/drivers/scipy_optimizer.py

```diff
@@ -177,6 +177,24 @@ def _declare_options(self):
         self.options.declare('singular_jac_tol', default=1e-16,
                              desc='Tolerance for zero row/column check.')
 
+    def _setup_comm(self, comm):
```

Comment:
This function isn't necessary.  Same goes for self._problem_comm.  All drivers have access to self._problem().comm so you should just use that.